ts/nni_manager/training_service/reusable/shared_storages/nfsStorageService.ts

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+'use strict';
+
+import * as cpp from 'child-process-promise';
+import * as path from 'path';
+
+import { SharedStorageService, SharedStorageConfig, SharedStorageType, LocalMountedType } from '../sharedStorage'
+import { MountedStorageService } from '../storages/mountedStorageService';
+import { TrialConfigMetadataKey } from '../../common/trialConfigMetadataKey';
+import { getLogger, Logger } from '../../../common/log';
+import { getExperimentId } from '../../../common/experimentStartupInfo';
+
+const INSTALL_NFS_CLIENT = `
+#!/bin/bash
+if [ -n "$(command -v apt-get)" ]
+then
+    sudo apt-get update
```

Comment:
If user's os already installed NFS, these bash script does not need to be executed.

Reply:
Add a `if [ -n "$(command -v nfsstat)" ]` judgment